scripts/secure-boot/generate-images/intermediate.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
@WiktorG351 would be nice if we would have one script with functions that allows to prepare needed image based on the provided arguments